routers/repo/setting_protected_branch.go

```diff
@@ -245,6 +245,7 @@ func SettingsProtectedBranchPost(ctx *context.Context, f auth.ProtectBranchForm)
 			}
 		}
 		protectBranch.BlockOnRejectedReviews = f.BlockOnRejectedReviews
+		protectBranch.DismissStaleApprovals = f.DismissStaleApprovals
```

Comment:
Shouldn't a change here from `false` to `true` dismiss existing stale reviews?

Reply:
Stale reviews are calculated regardless of the dismiss setting. The hourglass is also displayed on reviews marked stale. The dismissed setting only affect   which reviews count for mergability and is simply to add "stale=false" in the query.